src/server/pps/cmds/cmds.go

```diff
@@ -512,13 +512,25 @@ All jobs created by a pipeline will create commits in the pipeline's output repo
 			if err != nil {
 				return err
 			}
+			if jobID != "" {
+				ji, err := client.InspectJob(jobID, false)
+				if err != nil {
+					return err
+				}
+				jci, err := client.InspectCommit(ji.OutputCommit.Repo.Name, ji.OutputCommit.ID)
+				if err != nil {
+					return err
+				}
+				prov = append(prov, jci.Provenance...)
+			}
```

Comment:
Do this server-side instead, it's liable to be something we want to be able to call from other clients like the dash. Also it'll allow you to write a test case for it.